src/NuGet.Protocol.Catalog/Models/PackageDetailsCatalogLeaf.cs

```diff
@@ -78,5 +78,8 @@ public class PackageDetailsCatalogLeaf : CatalogLeaf
 
         [JsonProperty("verbatimVersion")]
         public string VerbatimVersion { get; set; }
+
+        [JsonProperty("icon")]
+        public string IconFilename { get; set; }
```

Comment:
Why isn't this `IconFileName`?

Reply:
Because that's a single [word](https://en.wikipedia.org/wiki/Filename)? :)

Reply:
Oh, TIL. Is this consistent across our docs/client code?

Reply:
Renamed to `iconFile` to be consistent with catalog entry.